harness/determined/cli/workspace.py

```diff
@@ -473,6 +477,8 @@ def yaml_file_arg(val: str) -> Any:
                             cli.Arg("workspace_name", type=str, help="name of the workspace"),
                             cli.Arg("cluster_name", type=str, help="cluster within which we create \
```

Comment:
In #9226, I added `clusterName` to the resource manager section of the master config to make the naming more intuitive.
Personally, I am not a _huge_ fan of changing the CLI argument to `resource-manager` because it implies (IMO) that the namespace's existence is tied to the Determined resource manager when it is actually tied to the cluster. 
I am open to more discussion about this though, because even `cluster-name` might get confusing since users might think this alludes to the name of the whole Determined deployment/cluster, rather than the `clusterName` in the resource manager's metadata in the master config

Reply:
> Personally, I am not a huge fan of changing the CLI argument to resource-manager because it implies (IMO) that the namespace's existence is tied to the Determined resource manager when it is actually tied to the cluster.

I thought that the resource manager is equivalent to the cluster(k8s cluster) in this project?

Reply:
They are equivalent as far as the way we handle them in Determined, but let's say that someone starts up a Kubernetes cluster with Minikube and names it `minikube_cluster_1`, and then names the resource manager `rm_1`. Then, they create a namespace in `minikube_cluster_1`. After this, they take down their master, start a new Minikube cluster named `minikube_cluster_2`, leaving the resource manager name the same (`rm_1`). Now, this new cluster does not have the namespace created in `minikube_cluster_1`, but now in the db, we would have a workspace-namespace binding where `resource_manager_name == rm_1` and namespace name is the name of the namespace created in  `minikube_cluster_1`, but this namespace no longer exists in `minikube_cluster_2`. 
Alternatively, if we use `clusterName` in the master config, the user would likely intuitively change `clusterName` to the name of the Minikube cluster, rather than leaving the resource manager name the same across different clusters. 
All this to say, I think `clusterName` makes more sense as far as the config goes, because it creates less opportunity for a disconnect on our side between resources associated with a given cluster and how we handle them in the db.

Reply:
Update: we have decided to change `Name` to `ClusterName` in the master config, so the argument name `--cluster-name` will be intuitive